code/src/components/Album.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Artists from './Artists'
+
+const Album = (props) => {
+  return props.album.map((album) => {
```

Comment:
You ware instructed to create a layout: 4/row-desktop, 2/row-tabler, 1/row-mobile. Please fix your layout to the given parameters.